### PR TITLE
Task 2.1, 2.2

### PR DESCRIPTION
**Pull request by Emidiant**
Найденные параметры обучения: 
learning_rate_decay = 0.8 
learning_rates = 0.1 
reg_strength = 1e-05

Лучшая точность на тренировочных данных у модели: 0.775
Neural net test set accuracy: 0.737

### CODE REVIEW

**Comment by Emidiant** [comment]
Reviewers @toshiks

**Comment by Emidiant** [comment]
Точность на тестовых данных составила 0.8192, при подобранных параметрах 
learning_rate = 1e-4, weight_decay = 1e-2, batch_size = 150, num_epoch = 30
Лучшая точность на тренировочных данных: 0.8407 
Использовался оптимизатор Adam.
